sound/hda/intel-nhlt.c

```diff
@@ -255,8 +255,12 @@ nhlt_get_specific_cfg(struct device *dev, struct nhlt_fmt *fmt, u8 num_ch,
 		dev_dbg(dev, "Endpoint format: ch=%d fmt=%d/%d rate=%d\n",
 			wfmt->channels, _vbps, _bps, wfmt->samples_per_sec);
 
+		/*
+		 * When looking for exact match of configuration ignore the vbps
+		 * from NHTL table when vbps == 0 passed to the function
```

Comment:
the comment is no longer valid, 'vbs' is ignored when 'ignore_vbs' is set

Reply:
gash, just a sec.